benchmark/scenarios/_rate_measurer.py

```diff
@@ -30,23 +36,25 @@ def request_sent(self):
         """
         self._sent += 1
 
-    def response_received(self, ignored):
+    def response_received(self, calltime):
         """
         Increase the number of received requests.
 
-        :param ignored: The result of a callback. This parameter is
-            not used.
+        :param float calltime: Time to perform call
         """
         self._received += 1
+        calltime = int(calltime)
```

Comment:
The nice thing about integers is that it gives a shape to the raw data that can be quickly scanned and basic stats can be performed in your head. As a compromise, I'm changing it to 1 decimal place.